client/src/main/kotlin/no/nav/su/se/bakover/client/oppdrag/utbetaling/UtbetalingRequest.kt

```diff
@@ -114,6 +115,14 @@ data class UtbetalingRequest(
             override fun toString() = value
         }
 
+        enum class KodeStatusLinje(@JsonValue val value: String) {
+            NY("NY"),
+            OPPHØR("OPPH"),
+            REAKKTIVER("REAK");
```

Comment:
REAKTIVER*, men bør den kanskje være REAKTIVER_OPPHØR, siden det sannsynligvis kommer REAKTIVER_HVIL, eller noe i den duren, senere?

Reply:
Var ikke klar over at denne eksisterte, tror like gjerne vi kan fjerne denne inntil vi tenker å innføre noe funksjonalitet knyttet til den. Etterhvert kan/bør vi vurder muligheten for å erstatte stans/gjenoppta med opphør/hvil/reak, da dette i mine øyne vil gi oss en mer helhetlig løsning til hvordan vi håndterer dette. Reaktivering fungerer for øvrig både på hvil og opphør.

Reply:
Aha, blir bare å bytte fra `REAKKTIVER` til `REAKTIVER` da.